website/app/styles/pages/application/footer.scss

```diff
@@ -54,7 +54,6 @@
       text-decoration: underline;
     }
 
-    // `ember-body-class` sets specific classes on the `<body>` element and we use this to have a full black background even in case of overscrolling
     body.application.index & {
```

Comment:
question: I wonder if we should be setting body styles in the footer component?

Reply:
we're not setting a body-style, is a parent selector